docs/deploying.md

```diff
@@ -0,0 +1,9 @@
+# Deploying
+
+See [the central documentation about merging and deploying](https://github.com/alphagov/notifications-manuals/wiki/Merging-and-deploying#deployment).
+
+## Caveats
+
+### The [deployment pipeline](https://github.com/alphagov/notifications-broadcasts-infra/blob/main/ci/govuk-alerts.yml) does't run functional tests for creating / approving alerts.
+
+You should use a branch deployment to manually test any changes that might affect the way the Celery worker runs or consumes from its queue e.g. changing to environment variables.
```

Comment:
I don't think the lack of functional test coverage should mean the suggested action should be to do a branch deployment. I'd suggest that instead it would be to deploy it in our normal way but pause the staging pipeline as per the wiki so that you can run some manual testing before unpausing the staging pipeline when you are happy.

I could see an argument instead that if you can't test this locally with all the apps working together (can't remember if you can or can't) then you'd do the branch deployment.

Reply:
That's fair - I should have said "some kind of manual testing". I've reworded to explain. I don't think we should encourage "merge and hope", as that can unnecessarily block the master branch - it's quick enough to do a branch deployment, or a local test. I've suggested all three approaches.

(I'm pretty sure the local Celery approach works, but I can't test.)

Reply:
Thanks, that is indeed an improvement